### DISCUSSION TITLE
Setup jdk with aarch64 architecture on a MX Mac

### DISCUSSION THREAD

**Discussion by dlagnohed** [coursier/coursier#2889] [Q&A]
When running the command `cs java --jvm corretto:21` for instance, Coursier only tries to install x64 binaries on my M1 Mac. How can I modify that command to make Coursier pick the aarch version of corretto:21 ?

**Comment by tgodzik** [comment]
IT should install the correct version for you architecture, I don't think you can define it explicitely.

**Reply by dlagnohed** [reply]
I'm afraid it does not. I managed to figure out a way to be explicit 18  months ago when I installed jdk18 on my machine but I've forgotten how I did that :(
